Squeezer/build.gradle

```diff
@@ -39,7 +39,7 @@ dependencies {
     compile 'com.google.code.findbugs:jsr305:2.0.2'
 
     // EventBus, https://github.com/greenrobot/EventBus.
-    compile project(':libs:eventbus:EventBus')
+    compile 'de.greenrobot:eventbus:2.4.1'
```

Comment:
I don't know that 2.4.1 actually fixes this.

Ahead of releasing 1.4.1 I've been launching Squeezer on some emulators running API 7, and they crash with

05-15 18:27:25.134 3064-3064/uk.org.ngo.squeezer W/System.err: java.lang.NoSuchMethodException
05-15 18:27:25.144 3064-3064/uk.org.ngo.squeezer W/System.err:     at java.lang.Class.getDeclaredMethods(Native Method)
05-15 18:27:25.154 3064-3064/uk.org.ngo.squeezer W/System.err:     at java.lang.ClassCache.getDeclaredMethods(ClassCache.java:153)
05-15 18:27:25.154 3064-3064/uk.org.ngo.squeezer W/System.err:     at java.lang.Class.getDeclaredMethods(Class.java:792)
05-15 18:27:25.154 3064-3064/uk.org.ngo.squeezer W/System.err:     at a.b.a.o.a(SubscriberMethodFinder.java:76)
05-15 18:27:25.165 3064-3064/uk.org.ngo.squeezer W/System.err:     at a.b.a.c.register(EventBus.java:164)
05-15 18:27:25.165 3064-3064/uk.org.ngo.squeezer W/System.err:     at a.b.a.c.register(EventBus.java:144)
05-15 18:27:25.165 3064-3064/uk.org.ngo.squeezer W/System.err:     at uk.org.ngo.squeezer.service.SqueezeService$EventBus.register(SqueezeService.java:1691)
05-15 18:27:25.165 3064-3064/uk.org.ngo.squeezer W/System.err:     at uk.org.ngo.squeezer.service.SqueezeService.onCreate(SqueezeService.java:220)
05-15 18:27:25.185 3064-3064/uk.org.ngo.squeezer W/System.err:     at android.app.ActivityThread.handleCreateService(ActivityThread.java:2780)
05-15 18:27:25.185 3064-3064/uk.org.ngo.squeezer W/System.err:     at android.app.ActivityThread.access$3200(ActivityThread.java:119)
05-15 18:27:25.185 3064-3064/uk.org.ngo.squeezer W/System.err:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1917)
05-15 18:27:25.185 3064-3064/uk.org.ngo.squeezer W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:99)
05-15 18:27:25.185 3064-3064/uk.org.ngo.squeezer W/System.err:     at android.os.Looper.loop(Looper.java:123)
05-15 18:27:25.195 3064-3064/uk.org.ngo.squeezer W/System.err:     at android.app.ActivityThread.main(ActivityThread.java:4363)
05-15 18:27:25.195 3064-3064/uk.org.ngo.squeezer W/System.err:     at java.lang.reflect.Method.invokeNative(Native Method)
05-15 18:27:25.195 3064-3064/uk.org.ngo.squeezer W/System.err:     at java.lang.reflect.Method.invoke(Method.java:521)
05-15 18:27:25.195 3064-3064/uk.org.ngo.squeezer W/System.err:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:860)
05-15 18:27:25.205 3064-3064/uk.org.ngo.squeezer W/System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)
05-15 18:27:25.205 3064-3064/uk.org.ngo.squeezer W/System.err:     at dalvik.system.NativeStart.main(Native Method)